webdriver-spec.html

```diff
@@ -2410,7 +2415,8 @@
 }</pre>
 </aside>
 
-<p>The <a>remote end steps</a> are:
+<p>The <a>remote end steps</a> <a data-lt="RES-arguments">with arguments</a> <var>url variables</var>
+ and <var>parameters</var> are:
```

Comment:
I don’t have strong feelings about whether we do or don’t change this. Do what you think is right.